cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/13-school-results.html

```diff
@@ -0,0 +1,91 @@
+<section class="college-costs_tool-section school-results" data-tool-section="school-results">
+    <h2>Compare the school to others</h2>
+
+    <div class="school-results_grad-content" data-program-type-content="grad">
+        <p>Now that you understand how much debt you’re considering, make sure to research how well the school serves students. You’re investing a lot—make sure it pays off.</p>
+        <p>Ask the school to provide the metrics you care about, such as:</p>
+        <ul>
+            <li>On-time completion rate</li>
+            <li>Job placement rate</li>
+            <li>Loan default rate</li>
+            <li>Loan repayment rate</li>
+            <li>Persistence and retention rate</li>
+        </ul>
+        <p>You should consider asking multiple schools for this information, so you can compare this school’s results.</p>
+    </div>
+
+    <div class="school-results_undergrad-content" data-program-type-content="undergrad">
+        <p>Now that you understand how much debt you’re considering, let’s look at some statistics that indicate how well the school serves students. You’re investing a lot—make sure it pays off.</p>
+
+        <h3 class="header_border-bottom"><span data-school-item="name"></span></h3>
+        <div class="school-results_basics">
+            <div class="school-results_basics__half">
```

Comment:
Technically this should go on the parent container so it's:
```
<div class="school-results_basics school-results_basics__half">
            <div>
```

And then if you need to target that inner div, you either target it generically with `.school-results_basics__half > div { … }`, or you make it an element `<div class="school-results_basics_wrapper">` and target that from the modifier `.school-results_basics__half .school-results_basics_wrapper { … }`.

Reply:
These classes aren't being used at the moment, so I removed them in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c.